source/frontend/src/components/maps/leaflet/Layers/HighwayParcelsLayer.tsx

```diff
@@ -12,10 +12,10 @@ export const HighwayParcelsLayer: React.FunctionComponent = () => {
 
   // For now, lat/long properties in the highway list will not display on the map
   // Ignore properties without a valid boundary
-  const validParcels = useMemo<Feature<Geometry, ISS_ProvincialPublicHighway>[]>(
-    () => (mapFeatureData?.highwayPlanFeatures.features ?? []).filter(p => exists(p?.geometry)),
-    [mapFeatureData?.highwayPlanFeatures.features],
-  );
+  const validParcels = useMemo<Feature<Geometry, ISS_ProvincialPublicHighway>[]>(() => {
+    const features = mapFeatureData?.highwayPlanFeatures.features ?? [];
```

Comment:
If highwayPlanFeatures in null or undefined the '.features' part will still fail. I think that  is the current issue atm